src/agenda/index.js

```diff
@@ -123,12 +125,27 @@ export default class AgendaView extends Component {
     }
   };
 
-  enableCalendarScrolling() {
+  /**
+   * Opens/closes the calendar programmatically.
+   * @param {bool} open `true` when calendar should open. Otherwise `false`.
+   */
+  setCalendarOpen = (open) => {
```

Comment:
The comment is uneccessary if the method name was descriptive enough. How about `toggleCalendarPosition(open)`?

Reply:
Updated!